engine/src/main/java/org/hibernate/search/engine/spi/DocumentBuilderIndexedEntity.java

```diff
@@ -444,7 +444,7 @@ private void buildDocumentFields(Object instance,
 			ConversionContext conversionContext,
 			InstanceInitializer objectInitializer,
 			final float inheritedBoost,
-			boolean multiValued,
+			boolean multiValuedParent,
```

Comment:
Feel free to suggest a better name. I just meant to mention that it's not the property we're processing that is multi-valued, but its parent property.
What's important is that even though the parent is not multi-valued, the property we're processing still can be.